tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/node_exception_weird.pysnap

```diff
@@ -250,7 +250,7 @@ app-depth-max:
 --------------------------------------------------------------------------
 system:
   hash: "252dc79eb5653bf822e2684d90734cb8"
-  tree_label: "eventFromException | captureException | invokeClient | captureException | finalReturnValue | mockConstructor [as captureException] | withScope"
+  tree_label: "eventFromException | captureException | invokeClient | captureException | <anonymous> | finalReturnValue | <anonymous> | mockConstructor [as captureException] | withScope"
```

Comment:
Suche frames do not really provide any value. Yet, if we do group by `"<anonymous>"`, it's a good idea to show.

Reply:
we group by filename for those frames, would you hide them?

Reply:
Oh, I wasn't aware of that at all. In that case, it's indeed misleading. 

This is now showing the complicated case: We group by file name, but we can't show the entire path. What if something not in the base name differs?

Reply:
decision: show filename basename, fallback to function, will explain in code comments